teammapper-backend/src/map/controllers/maps.controller.ts

```diff
@@ -26,6 +26,8 @@ export default class MapsController {
     })
     if (!map) throw new NotFoundException()
 
+    await this.mapsService.updateLastAccessed(mapId)
```

Comment:
maybe refetch the map here, as this return will not include the lastAccess field on the map. Please also check that it is returned, as I think we will need it later on. E.g. the settings page for your own maps might use it to show the correct deletion date.

Reply:
I did it the other way around: Updated lastAccessed first, then exportMapToClient is guaranteed to return a freshly updated lastAccessed field